OpenSim/Simulation/Model/FunctionBasedBushingForce.h

```diff
@@ -25,7 +25,6 @@
 
 
 // INCLUDE
-#include <OpenSim/Simulation/osimSimulationDLL.h>
```

Comment:
I think it is okay to leave in headers like this. Otherwise, we are relying on other headers (like Force.h) to have included this. Doesn't matter too much. The important thing is that `OSIMSIMULATION_API` is defined in this header.